lib/standard/version.rb

```diff
@@ -1,3 +1,3 @@
 module Standard
-  VERSION = Gem::Version.new("1.4.0")
+  VERSION = Gem::Version.new("1.4.1")
```

Comment:
Generally, it's considered bad form to bump the version number of a library in a PR, as it's the maintainer's responsibility to manage what goes into each release, in addition to version control, packaging, and publishing of each release (which, if you see the git log of the `version.rb` file, you'll see we're very particular about). As a result, it can sometimes come across as pushy or forward to bump the version of a library in a PR (the implication being an imperative "cut a release as soon as this is merged just for this change")